zio/core/src/main/scala/tofu/zioInstances/package.scala

```diff
@@ -17,7 +17,13 @@ class ZioInstances1 extends ZioInstances2 {
   private[this] val rioTofuInstanceAny: RioTofuInstance[Any] = new RioTofuInstance
   final def rioTofuInstance[R]: RioTofuInstance[R]           = rioTofuInstanceAny.asInstanceOf[RioTofuInstance[R]]
 
-  private[this] val zioErrorsToInstanceAny: ZioTofuErrorsToInstance[Any, Any, Nothing] = new ZioTofuErrorsToInstance
+  private[this] implicit val containsInstanceAny: Contains[Nothing, Any] =
+    new Contains[Nothing, Any] {
+      def set(s: Nothing, b: Any): Nothing = s
+      def extract(s: Nothing): Any = s
+    }
+
+  private[this] def zioErrorsToInstanceAny: ZioTofuErrorsToInstance[Any, Any, Nothing] = new ZioTofuErrorsToInstance[Any, Any, Nothing]
   final def zioTofuErrorsToInstance[R, E, E1]: ZioTofuErrorsToInstance[R, E, E1] =
```

Comment:
This instance is not sound anymore.
You can not transform `ZIO[R, E1, A]` to `ZIO[R, E, A]` without any prior knowledge about how to map `E1` to `E`
I'm afraid trying to cast arbitrary `E1` to `Nothing` would bring `ClassCastException`

Reply:
I suppose best we can do here is a change `E1` to `Nothing` explicitly for that instance.

Reply:
@Odomontois, you mean `final def zioTofuErrorsToInstance[R, E]: ZioTofuErrorsToInstance[R, E, Nothing]`?